index.bs

```diff
@@ -2683,6 +2683,7 @@ WritableStream(<var>underlyingSink</var> = {}, { <var>size</var>, <var>highWater
      potential types in the future, without backward-compatibility concerns.</p>
   1. Set *this*.[[writableStreamController]] to ? Construct(`<a idl>WritableStreamDefaultController</a>`, « *this*,
      _underlyingSink_, _size_, _highWaterMark_ »).
+  1. Return ? WritableStreamDefaultControllerStart(*this*.[[writableStreamController]]).
```

Comment:
Sorry, again. This logic touches the slots encapsulated in the controller. So, it should be a part of the WritableStreamDefaultController interfaces. That said, it was placed incorrectly. It should be in the section containing abstract operations exposed to the WritableStream. Fixing.